packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-config/spark-config.constants.js

```diff
@@ -1,7 +1,15 @@
 export const ARGUMENTS_VALIDATION_PATTERN = /^[\w].*/;
 export const PYTHON_ENV_FILENAME = 'environment.yml';
+export const PYTHON_FILES_CONTENT_TYPES = ['text/x-python'];
```

Comment:
```suggestion
export const PYTHON_FILES_CONTENT_TYPE = 'text/x-python';
```

Reply:
Hi @y4nnL ,
I'd rather keep the constant as an array for two reasons:
- keep consistency with the JAVA_FILES_CONTENT_TYPES constant (same logic)
- we might want to allow more python files types to be selected

Reply:
IMO we should do what logic demands, hence the suggestion. If one day this scenario occurs, let's do those changes ?

Reply:
Change done here: https://github.com/ovh/manager/pull/7069/commits/f767af04feeb30add4d8b4cf45c9704320c3aa14